node/src/utils/ds.rs

```diff
@@ -0,0 +1,18 @@
+//! Datasize helper functions.
```

Comment:
I'd rather keep this short, because it will absolutely clutter structs otherwise, as the function names are already quite long. Do you think it's worth it, only to end up with

```
    #[data_size(with = datasize_helper::hash_map_fixed_size)]
```

instead of 

```
    #[data_size(with = ds::hash_map_fixed_size)]
```

?

Reply:
If you're sure this module will never be used beyond these two places, then I can live with it.  However I have a feeling it'll get extended?

Either way - I won't insist.  I'll leave the decision up to you :)

Reply:
The goal is actually the opposite and have as much functionality as possible of this to go upstream into `datasize-rs` proper.

I'm fine with `datasize::hash_map_fixed_size`, but I absolutely do not want a name alias here (`utils::datasize` vs `datasize`).

Reply:
I'll leave it for now, but I'll give you a refactoring voucher if it becomes a problem, redeemable until 2022 :)